nixos/modules/services/games/pterodactyl-wings.nix

```
@@ -0,0 +1,214 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.services.pterodactyl-wings;
+  settingsFormat = pkgs.formats.yaml {};
+in
+{
+  options.services.pterodactyl-wings = with lib; with lib.types; {
+    enable = mkEnableOption "the Pterodactyl wings server manager";
+    tokenFile = mkOption {
+      type = types.path;
+      description = ''
+        Path to the panel-generated token for this wings instance.
+      '';
+    };
+    settings = mkOption {
+      description = ''
+        Configuration options for the wings daemon.
+        Some required values must be generated from the controlling panel.
+        See [reference](https://github.com/pterodactyl/wings/blob/develop/config/config.go)
+        for all options.
+      '';
+      default = {};
+      example = {
+        remote = "https://example.com";
+        uuid = "e851b74c-2e88-4d5c-8c6e-381218c7119d";
+        token_id = "fdZUuYH0k4aa8OGJ";
+        api = {
+          host = "127.0.0.1";
+          port = 6868;
+          upload_limit = 1000;
+        };
+        system = {
+          sftp.bind_port = 6869;
+          root_directory = "/srv/pterodactyl";
```

```suggestion
          root_directory = "/var/lib/pterodactyl";
```
NixOS per Default places data under /var/lib/

---

I meant this as an example of how to override the directories, but will change it.